### PR TITLE
Apply earlier typo fixes to source files instead of their output

### LINKED ISSUE #804: Please fix bad merge from #801
Several edits were made to an autogenerated file which will be lost on update. Comments are inline on the https://github.com/docker/distribution/pull/801.

### PR DESCRIPTION
**Pull request by aaronlehmann**
Regenerate api.md. This seems to result in a few unrelated changes.

Fixes #804

@stevvooe @RichardScothern

### CODE REVIEW

File: docs/spec/api.md [hunk 1]
```diff
@@ -1022,7 +1024,6 @@ A list of methods and URIs are covered in the table below:
 | PUT | `/v2/<name>/blobs/uploads/<uuid>` | Blob Upload | Complete the upload specified by `uuid`, optionally appending the body as the final chunk. |
 | DELETE | `/v2/<name>/blobs/uploads/<uuid>` | Blob Upload | Cancel outstanding upload processes, releasing associated resources. If this is not called, the unfinished uploads will eventually timeout. |
 | GET | `/v2/_catalog` | Catalog | Retrieve a sorted, json list of repositories available in the registry. |
-| DELETE | `/v2/<name>/blobs/<digest>` | Blob delete | Delete the blob identified by `name` and `digest`|
```
Comment (stevvooe) [inline docs/spec/api.md]: This one is disconcerted. Did a merge break somewhere? Was this added as part of @RichardScothern's work?

Comment (aaronlehmann) [inline docs/spec/api.md]: This was added to api.md as part of 9c1dd6943991797ca7ea099c5c7efb1890b249af, but not to descriptors.go. It needs to be added to descriptors.go.

Comment (stevvooe) [inline docs/spec/api.md]: I see.

**Comment by RichardScothern** [comment]
This was covered in #805.  Closing.

**Pull request closed without merging**